scss/_toasts.scss

```diff
@@ -2,31 +2,30 @@
   max-width: $toast-max-width;
   @include font-size($toast-font-size);
   color: $toast-color;
+  pointer-events: auto;
   background-color: $toast-background-color;
   background-clip: padding-box;
   border: $toast-border-width solid $toast-border-color;
   box-shadow: $toast-box-shadow;
-  opacity: 0;
   @include border-radius($toast-border-radius);
 
-  &:not(:last-child) {
-    margin-bottom: $toast-padding-x;
-  }
-
-  &.showing {
-    opacity: 1;
-  }
-
-  &.show {
-    display: block;
-    opacity: 1;
+  &:not(.showing):not(.show) {
+    opacity: 0;
   }
 
   &.hide {
     display: none;
   }
 }
 
+.toast-container {
+  display: grid;
+  width: max-content;
+  max-width: 100%;
```

Comment:
Drop these two lines for now? We can add them later if we really need them, but I think we can let our toasts stay at their `max-width`.

```suggestion
```